mypy.ini

```diff
@@ -104,3 +104,6 @@ ignore_missing_imports = True
 
 [mypy-chex.*]
 ignore_missing_imports = True
+
+[mypy-terminators_test_data.*]
+ignore_missing_imports = True
```

Comment:
My mypy-foo isn't the best so just wanna check that this is just for the terminator test and everywhere else it does still check for missing imports?

Reply:
I think this should be the case, but will wait for someone else to confirm.